File: sdkwork-app-sdk-go/types/chat_session_vo.go
package types

// 聊天会话响应
type ChatSessionVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Name string `json:"name"`
	Description string `json:"description"`
	Type string `json:"type"`
	Status string `json:"status"`
	ModelId string `json:"modelId"`
	MessageCount int `json:"messageCount"`
	UnreadCount int `json:"unreadCount"`
	LastMessageTime string `json:"lastMessageTime"`
	LastMessageContent string `json:"lastMessageContent"`
	CreateTime string `json:"createTime"`
	UpdateTime string `json:"updateTime"`
}

File: sdkwork-app-sdk-csharp/Models/ChatSessionVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ChatSessionVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public string? ModelId { get; set; }
        public int? MessageCount { get; set; }
        public int? UnreadCount { get; set; }
        public string? LastMessageTime { get; set; }
        public string? LastMessageContent { get; set; }
        public string? CreateTime { get; set; }
        public string? UpdateTime { get; set; }
    }
}
